compiler/damlc/daml-stdlib-src/DA/Text.daml

```diff
@@ -61,7 +61,7 @@ implode : [Text] -> Text
 implode = primitive @"BEImplodeText"
 
 -- | Test for emptiness.
-isEmpty : Text -> Bool
+isEmpty : P.Text -> Bool
```

Comment:
Probably you don't want to keep this test case?

Reply:
Oh wow, yes, oops. Thank you :)